firecracker/Dockerfile

```
@@ -207,9 +209,18 @@ RUN SYSTEMD_OFFLINE=1 systemctl enable network-setup
 # Copy custom Firecracker kernel (extracted by shed during image conversion)
 COPY --from=kernel-builder /build/linux/vmlinux /boot/vmlinux
 
-# Shell customization - colored prompt showing shed name and path
+# Shell customization - colored prompt showing shed name, path, and git branch
 USER shed
-RUN echo 'export PS1="\\[\\033[1;36m\\][\\[\\033[1;32m\\]shed:\\${SHED_NAME:-dev}\\[\\033[1;36m\\]]\\[\\033[0m\\] \\[\\033[1;34m\\]\\w\\[\\033[0m\\]\\$ "' >> /home/shed/.bashrc
+RUN cat >> /home/shed/.bashrc << 'PROMPT_EOF'
+__git_prompt() {
```

Hadolint (2.14.0)

[error] unexpected '_'
expecting '#', '\', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, at least one space, or end of input

(DL1000)